http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1045,32 +1064,39 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
   private[service] def getSavableCluster(
     userEmail: WorkbenchEmail,
     cloudContext: CloudContext,
+    autopilot: Boolean,
     now: Instant
   ): Either[Throwable, SaveKubernetesCluster] = {
     val auditInfo = AuditInfo(userEmail, now, None, now)
 
+    val nodepoolStatus =
+      if (autopilot || cloudContext.cloudProvider == CloudProvider.Azure) NodepoolStatus.Running
+      else NodepoolStatus.Precreating
     val defaultNodepool = for {
       nodepoolName <- KubernetesNameUtils.getUniqueName(NodepoolName.apply)
     } yield DefaultNodepool(
       NodepoolLeoId(-1),
       clusterId = KubernetesClusterLeoId(-1),
       nodepoolName,
-      status =
-        if (cloudContext.cloudProvider == CloudProvider.Azure) NodepoolStatus.Running else NodepoolStatus.Precreating,
+      status = nodepoolStatus,
       auditInfo,
       machineType = config.leoKubernetesConfig.nodepoolConfig.defaultNodepoolConfig.machineType,
       numNodes = config.leoKubernetesConfig.nodepoolConfig.defaultNodepoolConfig.numNodes,
       autoscalingEnabled = config.leoKubernetesConfig.nodepoolConfig.defaultNodepoolConfig.autoscalingEnabled,
       autoscalingConfig = None
     )
 
+    // autopilot mode clusters are regional
+    val loc =
```

Comment:
nit `location`

Reply:
not sure abt this one cuz it'll look like
```
SaveKubernetesCluster(
      cloudContext = cloudContext,
      clusterName = defaultClusterName,
      location = loc,
```
a few lines below...I think Scala will allow us to use the same name, but I've had comments from others they don't like that

Reply:
Oh gotcha, I am used to that convention but that is me from Python talking, feel free to ignore then